src/sonic-config-engine/portconfig.py

```diff
@@ -96,6 +96,29 @@ def get_hwsku_file_name(hwsku=None, platform=None):
             return candidate
     return None
 
+
+def get_port_info_from_config_db(asic_name=None):
```

Comment:
@abdosi i would suggest refactoring of `get_port_config`, since the method with the similar logic has been introduced. Please also add some args description with `asic_name` examples (e.g., `asic0`, `asic1`, etc.)